spec/requests/users_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+# This spec was generated by rspec-rails when you ran the scaffold generator.
+# It demonstrates how one might use RSpec to test the controller code that
+# was generated by Rails when you ran the scaffold generator.
+#
+# It assumes that the implementation code is generated by the rails scaffold
+# generator. If you are using any extension libraries to generate different
+# controller code, this generated spec may or may not pass.
+#
+# It only uses APIs available in rails and/or rspec-rails. There are a number
+# of tools you can use to make these specs even more expressive, but we're
+# sticking to rails and rspec-rails APIs to keep things simple and stable.
+  # This should return the minimal set of attributes required to create a valid
+  # User. As you add validations to User, be sure to
+  # adjust the attributes here as well.
+
+RSpec.describe "/users", type: :request do
+    describe "Post /users" do
+        it "returns http success" do
+        post users_path
+
+        expect(response.status).to eq(200)
+        expect(response).to match_response_schema("user")
+        end  
+    end
+    
+    describe "Get /users" do
+        it "returns http success" do
+        get user_path(:user)
+
+        expect(response.status).to eq(200)
+        expect(response).to match_response_schema("user")
+        end  
+    end 
+
+    describe "Get /users tweets paggination" do 
+        it "return http success" do 
+            get user_path(:tweet, :page, page: 2)
+
+            expect(response.status).to eq(200)
+            expect(response).to match_response_schema("user")
+        end
+    end
```

Comment:
Your test failed because you did not set up data 

```
/users
  Get /users tweets paggination
    return http success (FAILED - 1)

Failures:

  1) /users Get /users tweets paggination return http success
     Failure/Error: @user = User.find(params[:id])
     
     ActiveRecord::RecordNotFound:
       Couldn't find User with 'id'=tweet
     # ./app/controllers/users_controller.rb:63:in `set_user'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/actiontext-7.0.6/lib/action_text/rendering.rb:20:in `with_renderer'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/actiontext-7.0.6/lib/action_text/engine.rb:69:in `block (4 levels) in <class:Engine>'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/rack-2.2.8/lib/rack/tempfile_reaper.rb:15:in `call'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/rack-2.2.8/lib/rack/etag.rb:27:in `call'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/rack-2.2.8/lib/rack/conditional_get.rb:27:in `call'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/rack-2.2.8/lib/rack/head.rb:12:in `call'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/rack-2.2.8/lib/rack/session/abstract/id.rb:266:in `context'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/rack-2.2.8/lib/rack/session/abstract/id.rb:260:in `call'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/railties-7.0.6/lib/rails/rack/logger.rb:40:in `call_app'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/railties-7.0.6/lib/rails/rack/logger.rb:25:in `block in call'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/railties-7.0.6/lib/rails/rack/logger.rb:25:in `call'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/rack-2.2.8/lib/rack/method_override.rb:24:in `call'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/rack-2.2.8/lib/rack/runtime.rb:22:in `call'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/rack-2.2.8/lib/rack/sendfile.rb:110:in `call'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/railties-7.0.6/lib/rails/engine.rb:530:in `call'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/rack-test-2.1.0/lib/rack/test.rb:360:in `process_request'
     # /home/dev0/.rvm/gems/ruby-3.1.2/gems/rack-test-2.1.0/lib/rack/test.rb:153:in `request'
     # ./spec/requests/users_spec.rb:39:in `block (3 levels) in <top (required)>'

Finished in 0.04714 seconds (files took 0.4703 seconds to load)
1 example, 1 failure
```